packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -574,7 +585,7 @@ export class Function extends FunctionBase {
   /**
    * The DLQ associated with this Lambda Function (this is an optional attribute).
    */
-  public readonly deadLetterQueue?: sqs.IQueue;
+  public readonly deadLetterQueue?: sqs.IQueue | sns.ITopic;
```

Comment:
this change should be reverted right?

Reply:
Yes it should. I will add a second property `public readonly deadLetterTopic?: sns.ITopic` though.